mypy/semanal.py

```diff
@@ -511,6 +517,25 @@ def allow_unbound_tvars_set(self) -> Iterator[None]:
         finally:
             self.allow_unbound_tvars = old
 
+    @contextmanager
+    def inside_except_star_block_set(
+        self, value: bool, enteringLoop: bool = False
```

Comment:
Mypy follows PEP 8 naming conventions, so `enteringLoop` should probably be named `entering_loop` or similar

quick refs:
* https://github.com/python/mypy/wiki/Code-Conventions
* https://peps.python.org/pep-0008/#function-and-variable-names